packages/v4/src/content/design-guidelines/styles/icons/icons.md

```diff
@@ -97,7 +97,7 @@ Visit our [colors page](/guidelines/colors) to learn more about icon colors.
 ## All icons
 PatternFly uses custom icons and selections from <a href="https://fontawesome.com/icons?d=gallery&m=free">Font Awesome Free</a>. PatternFly icons are mostly two dimensional and flat. Navigate to Font Awesome’s website to download SVGs of any additional ‘fa’ icons within their free set. Proper attribution should be given as outlined on the Font Awesome site.
 
-Click on any single icon in the table to download it as an SVG. Download all icon SVGs <a href="https://patternfly-org.s3.us-east-2.amazonaws.com/patternfly-icons.zip">here</a>.
+Click on any single icon in the table to download it as an SVG. Download all icon SVGs <a href="https://github.com/patternfly/patternfly-design/blob/master/resources/patternfly-icon-svgs/patternfly-icons.zip">here</a>.
```

Comment:
This links to the Github page where the file lives, that page includes a "Download" button to download the file.

We could also change the link directly to the Download button link, which would immediately download the file (similar to the current functionality on our site) rather than redirect the user to a Github page, here's that URL:
- https://github.com/patternfly/patternfly-design/raw/master/resources/patternfly-icon-svgs/patternfly-icons.zip

Reply:
Yeah, I wasn't sure about that link. Thanks! I'll update the PR :)